requirements.txt

```diff
@@ -29,3 +29,6 @@ black==21.5b2
 pre-commit==2.13.0
 flake8==3.9.2
 isort==5.9.1
+
+# for sentiment filter
+spacytextblob==3.0.1
```

Comment:
Please create a separate `requirements.txt` file under your folder, so to avoid disturbing the main `requirements.txt` :)